environment-dev.yml

```diff
@@ -2,19 +2,9 @@ name: landlab_dev
 channels:
   - conda-forge
 dependencies:
```

Comment:
@mcflugen These requirements are all listed in `setup.py` am I correct to thing that they only should be listed there and not here as well?